ChangeSet
  1.1587.1.1 04/04/27 17:42:25 moilanen@zippy.ltc.austin.ibm.com +1 -0
  Fix for not vmalloc'n space for the sequence number in rtas_log_buf.

  arch/ppc64/kernel/rtasd.c
    1.26 04/04/27 17:42:16 moilanen@zippy.ltc.austin.ibm.com +1 -1
    Fix for not vmalloc'n space for the sequence number in rtas_log_buf.

diff -Nru a/arch/ppc64/kernel/rtasd.c b/arch/ppc64/kernel/rtasd.c
--- a/arch/ppc64/kernel/rtasd.c	Wed Apr 28 09:16:08 2004
+++ b/arch/ppc64/kernel/rtasd.c	Wed Apr 28 09:16:08 2004
@@ -372,7 +372,7 @@
 	if (event_scan == RTAS_UNKNOWN_SERVICE || get_eventscan_parms() == -1)
 		goto error;
 
-	rtas_log_buf = vmalloc(rtas_error_log_max*LOG_NUMBER);
+	rtas_log_buf = vmalloc(rtas_error_log_buffer_max*LOG_NUMBER);
 	if (!rtas_log_buf) {
 		printk(KERN_ERR "rtasd: no memory\n");
 		goto error;
.........................................................................
# vim: syntax=diff

